src/schematools/db.py

```diff
@@ -49,18 +49,18 @@ def create_meta_table_data(engine, dataset_schema: DatasetSchema):
     session.add(dataset)
 
     for table_data in dataset_schema["tables"]:
-        table_content = {camel_case_to_snake(k): v for k, v in table_data.items() if k != "schema"}
+        table_content = {camel_case_to_snake(k): v for k, v in table_data.default.items() if k != "schema"}
```

Comment:
I know this is old code. Maybe we should eat our own dogfood and use our own camel-snake converting functions from `schematools.utils`.

Reply:
👍 Will change